.gitignore

```diff
@@ -3,6 +3,7 @@ public/
 data/service_checks/
 data/npm
 integrations_data
+.hugo_build.lock
```

Comment:
I'm not that familiar with this new file I'm wondering if it should be committed instead of ignored? like yarn lock?

Reply:
@davidejones it looks like this was introduced when we recently upgraded Hugo versions.  The release notes recommend to leave the file alone, and .gitignore if we dont want it in the repository: https://gohugo.io/news/0.89.0-relnotes/#notes

Reply:
@davidejones this is a new lock file as part of the latest Hugo version. It has been suggested to never commit this file -- https://github.com/unicef/inventory/commit/5f978c252202dedb6c97782380df9b6f6d7c56e1

Reply:
ok got it thanks!